doc/about.rst

```diff
@@ -184,318 +184,192 @@ The project would like to thank the following funders.
 
   .. div:: text-box
 
-    `:probabl. <https://probabl.ai>`_ employs Adrin Jalali, Arturo Amor,
+    `:probabl. <https://probabl.ai>`_ is managing the whole sponsorship program
+    and employs full-time core maintainers such as Adrin Jalali, Arturo Amor,
```

Comment:
```suggestion
    and employs the full-time core maintainers Adrin Jalali, Arturo Amor,
```
small nit. I think this list is exhaustive (includes everyone) so we don't need 'such as'.